src/near-rpc-provider.spec.ts

```diff
@@ -26,7 +26,26 @@ describe('NearRpcProvider', () => {
     it('should get the balance for the account', async () => {
       const balance = await provider.getBalance('blockcoders.testnet')
       expect(balance).to.be.instanceOf(BigNumber)
-      expect(balance.gt(BigNumber.from(0))).to.be.true
+    })
+  })
+
+  describe('getBlock', () => {
```

Comment:
we have to add a test when you pass null or undefined. If you do, that should return the latest block.

Reply:
as discussed, not possible with the current ethers interface